firejail/mutt.profile

```diff
@@ -22,6 +32,9 @@ noblacklist ~/.signature
 noblacklist ~/.bogofilter
 noblacklist ~/.msmtprc
 
+# Allow executing /usr/sbin/sendmail
+noblacklist /usr/sbin
```

Comment:
Confirmed, doesn't work because `noblacklist` must match the exact thing given to `blacklist`.